mirdata/download_utils.py

```diff
@@ -8,6 +8,7 @@
 import tarfile
```

Comment:
remember to take this file out when merging

Reply:
done